app/controllers/concerns/api/v2/taxonomies_controller.rb

```diff
@@ -76,6 +77,13 @@ def destroy
 
   private
 
+  def rename_config_template
+    if params[taxonomy_single.to_sym] && !params[taxonomy_single.to_sym][:config_template_ids].nil?
```

Comment:
Could you please modify `params[taxonomy_single.to_sym]` to `params[taxonomy_single]`? we don't need to convert it to symbol since params are HashWithIndifferentAccess. I know it's not related to this issue but I'd consider the same change everywhere in this controller as acceptable in this PR.